config/ncg.conf

```diff
@@ -8,30 +8,20 @@
 # Variables in curly brackets are environment
 # variables.
 
-SITENAME = ${SITE_NAME}
-MAIN_DB_FILE=/etc/ncg/ncg.localdb
-
-MYPROXY_SERVER=${MYPROXY_SERVER}
-NAGIOS_SERVER=nagios.my.com
-PROBES_TYPE=all
-
 # Set following option if you wish to receive
 # email alerts for Nagios internal checks
 # default root@localhost
 # NAGIOS_ADMIN=admins@email.com
 
-# Set following option if remote gLite UI node is
-# used.
-# NRPE_UI=nrpe.srce.hr
-
 # list of VOs for which you wish to run checks
 VO = dteam
 
 #
 # Some optional config
 #
-#BDII = bdii.srce.hr
-#GLITE_VERSION=3.1.0
```

Comment:
what do they mean? please explaun at least  ENABLE_UNICORE_PROBES why we need it and what is myproxy
NAGIOS_SERVER = nagios.fqdn
ENABLE_UNICORE_PROBES=1
MYPROXY_SERVER = myproxy.fqdn